src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <>
+      <article className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
```

Comment:
You need to get "id" from destruction  and (add data-id={todo.id} attribute to each .TodoInfo). That's why your test failed)